Fix: typing for custom args in stories

dvdanielamoitzi opened the pull request:
Closes #689 

### Developer Checklist (Definition of Done)

**Issue**

- [ ] All acceptance criteria from the issue are met
- [ ] Tested in latest Chrome/Firefox

**UI/UX/Vis**

- [ ] Requires UI/UX/Vis review
  - [ ] Reviewer(s) are notified (_tag assignees_)
  - [ ] Review has occurred (_link to notes_)
  - [ ] Feedback is included in this PR
  - [ ] Reviewer(s) approve of concept and design

**Code**

- [ ] Branch is up-to-date with the branch to be merged with, i.e., develop
- [ ] Code is cleaned up and formatted
- [ ] Unit tests are written (frontend/backend if applicable)
- [ ] Integration tests are written (if applicable)

**PR**

- [ ] Descriptive title for this pull request is provided (will be used for release notes later)
- [ ] Reviewer and assignees are defined
- [ ] Add type label (e.g., *bug*, *feature*) to this pull request
- [ ] Add release label (e.g., `release: minor`) to this PR following [semver](https://semver.org/)
- [ ] The PR is connected to the corresponding issue (via `Closes #...`)
- [ ] [Summary of changes](#summary-of-changes) is written


### Summary of changes

- Extracted the props of the EagerVis into an interface (VisProps) so that I can re-use it in the stories
- extended the Meta type by CustomArgs, where I included the pointCount
- Tested the stories and the MainApp, everything worked well

### Screenshots
**Barchart:**
![image](https://github.com/user-attachments/assets/0b67910f-6e22-411d-a6a1-ffca3950cb75)
**Heatmap:**
![image](https://github.com/user-attachments/assets/314420e5-16f3-4026-977e-cea29772dae8)
**Hexbin:**
![image](https://github.com/user-attachments/assets/1d97295b-2a0d-4e2d-b35d-c644745f6f97)


### Additional notes for the reviewer(s)
@puehringer I'm not sure if this has further indications on the vis component (maybe the typing?), as I extracted the interface from the function.


-  
Thanks for creating this pull request 🤗

dvlilialbrecht reviewed (commented):
Shouldn't the HeatmapRandom Meta also be updated to the MetaArgs?

puehringer commented on src/vis/stories/Vis/Bar/BarRandom.stories.tsx:
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 
 import { Meta, StoryObj } from '@storybook/react';
 
+import { VisProps } from '../../../EagerVis';
```
@dvdanielamoitzi this import will break code-splitting and will lead to dozens of MP being included in the bundle (of storybook). Consider a import type instead, or just use `Parameters<typeof Vis>[0] & CustomArgs` instead (with the latter, you could also revert all your EagerVis changes).

dv-usama-ansari replied:
@puehringer this problem might go unnoticed by many other devs and we might end up loading huge initial bundle. Could you please go ahead with this PR so that we avoid this situation altogether: https://github.com/datavisyn/visyn_scripts/pull/103?

puehringer replied:
@dv-usama-ansari there is still an open comment in the PR: https://github.com/datavisyn/visyn_scripts/pull/103/files#r1924307618

But I agree, this is such an easy to make error that I have implemented safety measures in such as `bundlesize`: https://github.com/datavisyn/aevidence/blob/develop/package.json#L72-L77

dvdanielamoitzi commented:
Thanks for your review and inputs @puehringer @dv-usama-ansari and @dvlilialbrecht . I am now using `Parameters<typeof Vis>[0]` and reverted the changes in EagerVis.
Also, I added it to the HeatmapRandomData.
